examples/qml_minimal/CMakeLists.txt

```diff
@@ -22,47 +16,55 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core Gui Qml QuickControls2 QmlImportScanner REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core Gui Qml QuickControls2 QmlImportScanner REQUIRED)
-
 # ANCHOR: book_cmake_generation
 # ANCHOR: book_tutorial_cmake_diff_1
-include(CxxQt)
-
-# Generate C++ code from Rust using Cargo in the current folder
-cxx_qt_generate_cpp(GEN_SOURCES)
-# ANCHOR_END: book_tutorial_cmake_diff_1
 
-# Define our sources
-set(
-    CPP_SOURCES
-    ${CMAKE_CURRENT_SOURCE_DIR}/src/main.cpp
-)
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Core Gui Qml QuickControls2 QmlImportScanner REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core Gui Qml QuickControls2 QmlImportScanner REQUIRED)
+# The path to the qmake executable path needs to be passed to the Rust library's build script to ensure
+# it uses the same installation of Qt as CMake.
+get_target_property(QMAKE Qt${QT_VERSION_MAJOR}::qmake IMPORTED_LOCATION)
+
+find_package(Corrosion)
```

Comment:
As on the next line if the package isn't found we then fetch it, i would prefer this not to warn (as it's a false warning). So i think put `find_package(Corrosion QUIET)` instead?